base/install_node

```diff
@@ -79,17 +79,22 @@ function verifyBinaryExists(version, cb) {
 
 
 function downloadAndInstallVersion(version, cb) {
-  var downloadUrl = 'https://storage.googleapis.com' +
-    '/gae_node_packages/node-' + version + '-linux-x64.tar.gz';
   var destPath = '/nodejs';
-  var cmd = 'curl ' + downloadUrl + '| tar xzf - -C ' + destPath +
-    ' --strip-components=1';
+  var baseUrl = 'https://storage.googleapis.com/gae_node_packages';
+  var cmd = '/opt/nodejs/verify_install_node ' + baseUrl +
+    ' ' + version + ' ' + version + '-SHASUMS256.txt.asc ' + destPath +
+    ' WARN_ON_ERROR';
 
-  exec('rm -rf ' + destPath + '/*', function(err) {
-    if (err) {
-      return cb(err);
-    }
-    exec(cmd, cb);
+  var proc = exec(cmd, { maxBuffer: 500*1024 }, function(error) {
```

Comment:
If the callback with stdout/stderr is used, then contents of stdout/stderr are only available after the invoked command has completed which can cause the installation to look as if it has hung.

From looking at the docs, it looks like care needs to be taken with `spawn` to invoke the callback only once since multiple 'error' events can be emitted with or without an 'exit' event event being emitted.